File: VB/XtraReport1.vb
Imports System
' ...

Namespace PageWidth
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub xrLabel1_BeforePrint(ByVal sender As Object, ByVal e As System.Drawing.Printing.PrintEventArgs) Handles xrLabel1.BeforePrint
			xrLabel1.Width = Convert.ToInt32(PrintingSystem.PageSettings.UsablePageSize.Width) - 1
		End Sub

	End Class
End Namespace

File: CS/XtraReport1.cs
using System;
// ...

namespace PageWidth {
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport {
        public XtraReport1() {
            InitializeComponent();
        }

        private void xrLabel1_BeforePrint(object sender, System.Drawing.Printing.PrintEventArgs e) {
            xrLabel1.Width = Convert.ToInt32(PrintingSystem.PageSettings.UsablePageSize.Width) - 1;
        }

    }
}
